src/components/PopupSelect/index.jsx

```diff
@@ -70,6 +70,8 @@ class PopupSelect extends Component {
     const { history } = this.state
     const current = history[0]
     const children = current.children || []
+    const selectedItem = children.find(this.props.isSelected)
```

Comment:
So here, we are "lucky" because the "others" category comes in last ? What makes this.props.isSelected be true for others ? Hasn't it be fixed by the previous commit ?

Reply:
Yes we are more or less lucky. The problem is that a category and its corresponding « others » subcategory have the same id. Since `this.props.isSelected` returns `true` for the parent category too, then it returns `true` for the « others » subcategory.